### PR TITLE
replace zson with zng

### PR DESCRIPTION
**Pull request by mccanne**

### CODE REVIEW

File: cmd/zq/zq.go [hunk 1]
```diff
@@ -92,8 +92,8 @@ type Command struct {
 func New(f *flag.FlagSet) (charm.Command, error) {
 	cwd, _ := os.Getwd()
 	c := &Command{dt: resolver.NewTable()}
-	f.StringVar(&c.ifmt, "i", "auto", "format of input data [auto,bzson,ndjson,zeek,zjson,zson]")
-	f.StringVar(&c.ofmt, "f", "zson", "format for output data [text,table,zeek,ndjson,bzson,zson]")
+	f.StringVar(&c.ifmt, "i", "auto", "format of input data [auto,bzng,ndjson,zeek,zjson,zng]")
+	f.StringVar(&c.ofmt, "f", "zng", "format for output data [text,table,zeek,ndjson,bzng,zng]")
```
Comment (nwt) [inline cmd/zq/zq.go]: Nit: Sort the list of output formats for readability.

File: cmd/zq/zq.go [hunk 2]
```diff
@@ -230,7 +230,7 @@ func (c *Command) loadFile(path string) (zson.Reader, error) {
 	switch c.ifmt {
 	case "auto":
 		return detector.NewReader(f, c.dt)
-	case "ndjson", "bzson", "zeek", "zjson", "zson":
+	case "ndjson", "bzng", "zeek", "zjson", "zng":
```
Comment (nwt) [inline cmd/zq/zq.go]: Nit: Maybe sort list for readability.

**Comment by mccanne** [comment]
> Some Markdown files still need an update, but that can wait.

Yes this is coming...

File: cmd/zq/zq.go (hunk 1 above)
Comment (mccanne) [reply cmd/zq/zq.go]: +1

File: cmd/zq/zq.go (hunk 2 above)
Comment (mccanne) [reply cmd/zq/zq.go]: +1

**Pull request merged**